package.json

```diff
@@ -37,66 +37,65 @@
     "@amsterdam/design-system-react": "0.11.1",
     "@amsterdam/design-system-tokens": "0.11.0",
     "@formio/react": "5.3.0",
-    "formiojs": "4.21.3",
-    "keycloak-js": "25.0.5",
-    "next": "14.2.10",
+    "formiojs": "4.21.4",
+    "keycloak-js": "25.0.6",
+    "next": "14.2.15",
     "ra-data-simple-rest": "5.1.4",
     "ra-i18n-polyglot": "5.1.4",
     "ra-keycloak": "1.0.1",
-    "ra-language-dutch": "4.6.1",
     "react": "18.3.1",
     "react-admin": "5.1.4",
     "react-dom": "18.3.1",
     "tslib": "2.7.0"
   },
   "devDependencies": {
-    "@babel/core": "7.25.2",
-    "@babel/preset-react": "7.24.7",
-    "@hey-api/openapi-ts": "0.48.0",
-    "@nx/eslint": "19.7.2",
-    "@nx/eslint-plugin": "19.7.2",
-    "@nx/jest": "19.7.2",
-    "@nx/js": "19.7.2",
-    "@nx/next": "19.7.2",
-    "@nx/react": "19.7.2",
-    "@nx/vite": "19.7.2",
-    "@nx/web": "19.7.2",
-    "@nx/workspace": "19.7.2",
+    "@babel/core": "7.25.8",
+    "@babel/preset-react": "7.25.7",
+    "@hey-api/openapi-ts": "0.53.9",
```

Comment:
This will break `generate-api-client`, let's leave it for now

Reply:
You mean this line? "@hey-api/openapi-ts": "0.53.9",

Reply:
Yep